Source/control.cpp

```diff
@@ -1691,11 +1692,13 @@ void ReleaseChrBtns(bool addAllStatPoints)
 	}
 }
 
-static int DrawDurIcon4Item(CelOutputBuffer out, ItemStruct *pItem, int x, int c)
+#define DURABILITY_RED 2
+#define DURABILITY_GOLD 5

```

Comment:
Can we use `consts` for new code instead of `#define`s?

Reply:
Done.